gradle.properties

```diff
@@ -1,6 +1,15 @@
-kotlin.code.style=official
-org.gradle.jvmargs=-Xmx2g -XX:MaxMetaspaceSize=512m -XX:+HeapDumpOnOutOfMemoryError -Dfile.encoding=UTF-8
-org.gradle.caching=true
-# https://kotlinlang.org/docs/whatsnew19.html#preview-of-gradle-configuration-cache
-org.gradle.configuration-cache=false
-org.gradle.parallel=true
\ No newline at end of file
+#########################
+# Gradle settings
+#########################
+org.gradle.jvmargs = -Xmx2g -XX:MaxMetaspaceSize=512m -XX:+HeapDumpOnOutOfMemoryError -Dfile.encoding=UTF-8
+org.gradle.parallel = true
+org.gradle.caching = true
+org.gradle.configuration-cache = false
```

Comment:
Configuration cache is kept as disabled because of Reckon I guess.  
It uses Git under the hood and it messes up the cache compatibility.

Reply:
Sounds good to me!